docs/pull_request_template.md

```diff
@@ -0,0 +1,16 @@
+Closes #
+
+Changes proposed in this pull request:
+
+-
+-
+-
+
+Pull request checklist:
+
+- [ ] branch is based on `master`
+- [ ] proper commit messages
+- [ ] code is properly tested
```

Comment:
I wonder if this should be separated in two parts? Something like:

1. tested in a browser
2. wrote unit test